src/routes/auth/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, test } from 'vitest';
```

Comment:
I'm not sure how you were able to push this code I get this error locally:

```
Parsing error: Unknown compiler option 'erasableSyntaxOnly'.eslint

```

Reply:
Sorry, I've updated all of the packages/dependencies, so to run it okay reinstall them:

```shell
npm install --save-dev
```